apps/nextjs-website/react-components/components/MediaResources/MediaResources.tsx

```diff
@@ -0,0 +1,103 @@
+import { Box, Typography, Grid, Link, useTheme } from '@mui/material';
+import { MediaResourcesProps } from '@react-components/types';
+import Image from 'next/image';
+import {
+  SendBackgroundColor,
+  IoBackgroundColor,
+  TextColor,
+} from '../common/Common.helpers';
+
+const MediaResources = ({
+  items,
+  sectionID,
+  theme,
+  themeVariant,
+  sectionTitle,
+}: MediaResourcesProps) => {
+  const muiTheme = useTheme();
+
+  const backgroundColor =
+    themeVariant === 'SEND'
+      ? SendBackgroundColor(theme)
+      : IoBackgroundColor(theme);
+
+  const textColor = TextColor(theme);
+
+  const linkColor =
+    theme === 'dark'
+      ? muiTheme.palette.custom.white
+      : themeVariant === 'SEND'
+        ? muiTheme.palette.primary.main
+        : muiTheme.palette.custom.primaryColorDark;
+
+  return (
+    <Box
+      component='section'
+      id={sectionID || undefined}
```

Comment:
```suggestion
      {...(sectionID && { id: sectionID })}
```
Prefer not setting variables instead of setting them to undefined or null